ddl/ddl_api.go

```diff
@@ -4891,6 +4891,100 @@ func (d *ddl) DropIndex(ctx sessionctx.Context, ti ast.Ident, indexName model.CI
 	return errors.Trace(err)
 }
 
+func (d *ddl) DropIndexes(ctx sessionctx.Context, ti ast.Ident, specs []*ast.AlterTableSpec) error {
```

Comment:
Much code is duplicate with `DropIndex`, please refactor it.

Reply:
> Much code is duplicate with `DropIndex`, please refactor it.

Do you want me to replace DropInde with a DropIndex？